toolbelt/generate-configs.js

```diff
@@ -3,29 +3,43 @@
  */
 
 const fs = require( 'fs' );
-const packageJsonTemplate = require( './templates/package.json' );
+const path = require( 'path' );
 
 const packageList = require('./packages-list.js').packageList();
 
 console.log('Generating package.json and tsconfig.json for:');
 
+const packagesRootDir = path.resolve(__dirname, '..', 'packages');
+const templatesRootDir = path.resolve(__dirname, 'templates');
+
+const packageJsonTemplate = require( path.resolve(templatesRootDir, 'package.json') );
+const tsconfigJsonTemplate = require( path.resolve(templatesRootDir, 'tsconfig.json') );
+
 for (let packageName of packageList) {
 
   console.log(packageName);
 
-  fs.createReadStream('./toolbelt/templates/tsconfig.json').pipe(fs.createWriteStream(`./packages/${packageName}/tsconfig.json`));
+  const packageRootDir = path.resolve(packagesRootDir, packageName);
+
+  // Copy tsconfig.json template to each package folder
+  fs.writeFileSync(path.resolve(packageRootDir, 'tsconfig.json'), JSON.stringify(tsconfigJsonTemplate, null, 2) + "\n")
```

Comment:
It might look better, when you extract it to a named function, so that everyone can see what is going on.

Also, why do we need **"\n"** at the end - is it really necessary?